crates/cli/src/cli/mod.rs

```diff
@@ -254,17 +257,21 @@ impl StartSimnet {
     }
 
     pub fn simnet_config(&self, airdrop_addresses: Vec<Pubkey>) -> SimnetConfig {
-        let remote_rpc_url = match &self.network {
-            Some(NetworkType::Mainnet) => DEFAULT_RPC_URL.to_string(),
-            Some(NetworkType::Devnet) => DEVNET_RPC_URL.to_string(),
-            Some(NetworkType::Testnet) => TESTNET_RPC_URL.to_string(),
-            None => match self.rpc_url {
-                Some(ref rpc_url) => rpc_url.clone(),
-                None => match env::var("SURFPOOL_DATASOURCE_RPC_URL") {
-                    Ok(value) => value,
-                    _ => DEFAULT_RPC_URL.to_string(),
+        let remote_rpc_url = if self.offline {
```

Comment:
Instead of making offline an empty string, can we make the `remote_rpc_url` field an optional?